ts_commands_test.go

```diff
@@ -246,30 +247,52 @@ func TestBuildTsListKeysReqCorrectlyViaBuilder(t *testing.T) {
 }
 
 func TestNewTsCells(t *testing.T) {
+	s := int64(1234567890)
+	ms := time.Duration(103) * time.Millisecond
+	tval := time.Unix(s, ms.Nanoseconds())
+
 	cells := make([]TsCell, 5)
 	cells[0] = NewStringTsCell("Test Key Value")
 	cells[1] = NewSint64TsCell(1)
 	cells[2] = NewDoubleTsCell(0.1)
 	cells[3] = NewBooleanTsCell(true)
-	cells[4] = NewTimestampTsCell(1234567890)
+	cells[4] = NewTimestampTsCell(tval)
 
-	if expected, actual := "VARCHAR", cells[0].GetDataType(); expected != actual {
-		t.Errorf("expected %v, got %v", expected, actual)
+	if got, want := cells[0].GetDataType(), "VARCHAR"; got != want {
```

Comment:
FYI, `got, want` is what the golang stdlib uses. I've been converting tests as I go in this code.